scripts/run_benchmarks.sh

```diff
@@ -15,6 +15,7 @@ ALL_EXTERNAL_PALLETS=( \
   pallet_preimage \
   pallet_scheduler \
   pallet_session \
+  pallet_sudo \
```

Comment:
Per other comment, I think we can skip this, but also if it works, then leave it.